templates/tests/test-templates.sh

```diff
@@ -116,6 +120,11 @@ function deployTemplate {
 # $2 - The branch name
 # $3 - The environment name
 function testTemplate {
+    if [ $VALIDATE == false ]; then 
+        echo "Skipping playwright validation for $1..."
+        return
```

Comment:
Do we know an idea of what the effort in how we can validate AKS in the long run? It is possible that we run some `az` update commands to poke through the firewall for this particular test setup?

These are non-prod resources, and it seems like we're just subjected to the default network policies that are more restrictive than what we intend for. Would be nice to have a work item capturing this so we don't forget.

Reply:
Logging internal issue @ https://github.com/Azure/azure-dev-pr/issues/1608